.gitmodules

```diff
@@ -69,7 +69,7 @@
 	sbom-supplier = Organization: fmt <https://fmt.dev/latest/index.html>
 	sbom-url = https://github.com/fmtlib/fmt/
 	sbom-description = A modern formatting library
-	sbom-hash = a33701196adfad74917046096bf5a2aa0ab0bb50
+	sbom-hash = e57ca2e3685b160617d3d95fcd9e789c4e06ca88
```

Comment:
Please change also the sbom-version, 5 lines above

Reply:
Missed that, thanks.